src/zorak/cogs/general/general_trans.py

```diff
@@ -1,74 +1,81 @@
+
 import discord.ext
 import googletrans
 from discord.ext import commands
 from googletrans import Translator
 
+
+def is_multi_lang(lang):
+    if isinstance(lang, list):
+        return True
+    else:
+        return False
+
+
 class googletrans_func(commands.Cog):
```

Comment:
I would just name it:
`class GoogleTranslate(commands.Cog)`

by PEP 8, classes should use CapsWord convention and not include underscores

Reply:
Indeed, and changed!